### ISSUE TITLE
how to keep img tag

### ISSUE THREAD

**Issue by ohblue** [matthiask/html-sanitizer#34]
I have allowed the <img> tag by seeting 
```
Sanitizer({
    "tags": ("a", "h1", "h2", "h3", "strong", "em", "p", "ul", "ol","li", "br", "sub", "sup", "hr")
})
```
but the <img> tag is still being removed.

**Comment by matthiask** [comment]
Hi @ohblue 

I added a test which demonstrates how you can preserve the `img` tag. See https://github.com/matthiask/html-sanitizer/commit/74a7a595790bc974448124add8ba66630f663bd4

**Comment by seamys** [comment]
@matthiask  No, that should not work, because you have completely overwride exists setting

**Comment by seamys** [comment]
`
# Make a copy
html_settings = dict(html_sanitizer.sanitizer.DEFAULT_SETTINGS)

# Add your changes
html_settings['tags'].add('img')
html_settings['empty'].add('img')
html_settings['attributes'].update({'img': ('src', 'alt', 'title')})

sanitizer = Sanitizer(settings=html_settings)

`
